### PR TITLE
mock-array: 2000 to 1000ps

### PR DESCRIPTION
**Pull request by oharboe**
@nerriav @efis1 @louiic Does this look OK?

I have been learned a little bit more about hold cells and times in the last couple of days while reporting on various issues in OpenROAD.

### CODE REVIEW

**Comment by maliberty** [comment]
CI fails

**Comment by oharboe** [comment]
> CI fails

Will do `make update_ok` if it passes review otherwise

File: flow/designs/asap7/mock-array/metadata-base-ok.json [hunk 1]
```diff
@@ -15,59 +15,59 @@
...
   "cts__design__instance__area__post_repair": 119918,
   "cts__design__instance__area__pre_repair": 119918,
-  "cts__design__instance__area__stdcell": 511.408,
+  "cts__design__instance__area__stdcell": 1899.57,
   "cts__design__instance__area__stdcell__post_repair": 478.968,
   "cts__design__instance__area__stdcell__pre_repair": 478.647,
-  "cts__design__instance__count": 10619,
-  "cts__design__instance__count__hold_buffer": 445.0,
+  "cts__design__instance__count": 29661,
+  "cts__design__instance__count__hold_buffer": 19487.0,
```
Comment (maliberty) [inline flow/designs/asap7/mock-array/metadata-base-ok.json:28]: It's interesting that this went up so much...

**Review by maliberty — APPROVED** [review]

File: flow/designs/asap7/mock-array/metadata-base-ok.json (hunk 1 above)
Comment (oharboe) [reply flow/designs/asap7/mock-array/metadata-base-ok.json:28]: LOTS of hold cells at the edges of mock-array. I wonder if carefully articulated .sdc files can do 500ps...

I wonder if reducing https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/077cb25732f7162ac688f0944aa5020d32c5af42/flow/designs/asap7/mock-array/constraints.sdc#L8 would improve things.

**Pull request merged**